src/notebook_unit1.ipynb

```diff
@@ -220,38 +224,9 @@
     }
    ],
    "source": [
-    "print(thershv)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 17,
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "0      0.0\n",
-       "1      0.0\n",
-       "2     -0.0\n",
-       "3     -0.0\n",
-       "4     -0.0\n",
-       "      ... \n",
```

Comment:
It seems like for Jings commits the nbstripout hook doesn't work, as output is still in the .ipnb file.  Not sure if this is a problem though, because we will switch to IDEs instead of Jupyter in the next tasks.